### PR TITLE
feature/show_event_day_view

### PR DESCRIPTION
**Pull request by ADI-projects30**
1. When press on button "event details" the event view will be load into day view.
2. When press on plus icon , redirect to create event

### CODE REVIEW

**Review by yammesicka — CHANGES_REQUESTED** [review]
Great job! I've added few of my insights, please take a look :)

File: app/static/dayview.css [hunk 1]
```diff
@@ -101,4 +101,13 @@ html {
     background-color: var(--borders-variant);
     border-radius:50px;
     box-shadow: 1px 1px 2px #999;
+}
+
+.event-btn {
+    font-size: 10px;
```
[the remark this answers is not in the document]
Comment (ADI-projects30) [reply app/static/dayview.css]: fixed all of the notes:, waiting for merge :)

File: app/static/eventdisplay.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    ev.addEventListener("click", displayEvent);
+}
+
+function displayEvent(wantedevent) {
+    fetch('/event/' + wantedevent)
+    .then(function(response) {
+        return response.text();
+    })
+    .then(function(body) {
+        document.querySelector('#here').innerHTML = body;
```
[the remark this answers is not in the document]
Comment (ADI-projects30) [reply app/static/eventdisplay.js]: tried to do it with innerText but it didn't work, so i stayed with innerHTML

File: app/static/eventdisplay.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+function scrollTo(element) {
+    window.scroll({
+      behavior: 'smooth',
+      left: 0,
+      top: element.offsetTop
+    });
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+    let chosen_button = document.getElementsByClassName('event-btn');
```
Comment (yammesicka) [inline app/static/eventdisplay.js:10]: Can be `const`

File: app/templates/dayview.html [hunk 4]
```diff
@@ -53,12 +53,19 @@
                 
                 {% for event, attr in events %}
                     <div class="d-flex flex-row justify-content-around align-items-end action-continer" style="grid-row: {{attr.grid_position}};">
-                        <a href="/edit/{{event.id}}" title="Edit event" class="action-icon"><img class="pb-1" src="{{ url_for('static', path='/images/icons/pencil.svg')}}" width="15em" height="15em"></a>
-                        <a href="/delete/{{event.id}}" title="Delete event" class="action-icon"><img class="pb-1" src="{{ url_for('static', path='/images/icons/trash-can.svg')}}" width="15em" height="15em"></a>
+                        <a href="/edit/{{event.id}}" title="Edit event" class="action-icon"><img class="pb-1" src="{{ url_for('static', path='/images/icons/pencil.svg')}}"></a>
+                        <a href="/delete/{{event.id}}" title="Delete event" class="action-icon"><img class="pb-1" src="{{ url_for('static', path='/images/icons/trash-can.svg')}}"></a>
+                        <button class="event-btn" title="See full event" id="{{event.id}}">event details</button>
```
Comment (yammesicka) [inline app/templates/dayview.html:58]: I think it should be the other way around - the a should be a button, and the "See full event" should be a link :)

File: app/templates/dayview.html [hunk 5]
```diff
@@ -53,12 +53,19 @@
...
                     </div>
                 {% endfor %}
             </div>
         </div>
+        <div>
+            <a href="/event/edit" title="Add Event" class="add-event-icon"><img class="plus_image" src="{{ url_for('static', path='/images/icons/plus.svg')}}"></a>
+        </div>
+        <div id="here"></div>
+        <div id="nice">Have a nice day</div>
```
Comment (yammesicka) [inline app/templates/dayview.html:67]: ?

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: app/static/eventdisplay.js (hunk 3 above)
Comment (ADI-projects30) [reply app/static/eventdisplay.js]: fixed all of the notes:, waiting for merge :)

**Review by yammesicka — APPROVED** [review]

**Pull request merged**